tests/unit/normalize_test.py

```diff
@@ -0,0 +1,176 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Tests normalization scheme for normalizing detectors' scores according
+to baseline."""
+
+import csv
+import json
+import os
+import shutil
+import tempfile
+import unittest
+
+from nab.runner import Runner
+
+
+def createTemporaryCsv(parentDir, fileName, data):
```

Comment:
change name b/c there's nothing about this method that requires the csv (or its dir) to be temporary